server_src/runners/state.js

```diff
@@ -2546,23 +2546,23 @@ class Runner {
     return ClientAPI.getRetry(
       farmPrinters[index].printerURL,
       farmPrinters[index].apikey,
-      "api/plugin/pluginmanager"
+      "plugin/pluginmanager/plugins"
```

Comment:
What about previous versions?

Reply:
I guess we'll have to support both, because the API will become obsolete one day. Shall I do a version check here?

Reply:
We're going to have to, it would break functionality I gather pre 1.6.0 and we know how fast famers update xD